apps/staking/package.json

```diff
@@ -24,35 +24,36 @@
     "@session/util-js": "workspace:*",
     "@session/util-logger": "workspace:*",
     "@session/wallet": "workspace:*",
-    "@tanstack/react-query": "5.51.1",
+    "@tanstack/react-query": "5.61.5",
     "better-sqlite3-multiple-ciphers": "11.1.2",
+    "bufferutil": "^4.0.8",
     "class-variance-authority": "^0.7.0",
     "date-fns": "^3.6.0",
-    "next": "14.2.12",
-    "next-intl": "^3.14.1",
+    "next": "15.0.4",
+    "next-intl": "3.26.0",
     "pino": "^9.2.0",
     "pino-pretty": "^11.2.1",
-    "react": "18.3.1",
-    "react-dom": "18.3.1",
-    "react-hook-form": "^7.51.5",
+    "react": "19.0.0-rc-66855b96-20241106",
+    "react-dom": "19.0.0-rc-66855b96-20241106",
+    "react-hook-form": "7.53.2",
     "rtl-detect": "^1.1.2",
     "server-only": "^0.0.1",
     "sharp": "0.32.6",
     "sqids": "^0.3.0",
-    "wagmi": "2.12.25",
+    "utf-8-validate": "^6.0.5",
+    "viem": "2.21.54",
     "zod": "^3.23.8"
   },
   "devDependencies": {
-    "@next/eslint-plugin-next": "^14.1.1",
+    "@next/eslint-plugin-next": "15.0.3",
     "@session/eslint-config": "workspace:*",
     "@session/typescript-config": "workspace:*",
-    "@types/react": "^18.2.61",
-    "@types/react-dom": "^18.2.19",
+    "@types/react": "npm:types-react@19.0.0-rc.1",
```

Comment:
Interesting that you have to specify npm here for the package value.

Reply:
Welcome to the world of multiple registries https://jsr.io/